extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/CacheManagerImpl.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.cache.runtime;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import io.quarkus.cache.Cache;
+import io.quarkus.cache.CacheManager;
+
+@ApplicationScoped
+public class CacheManagerImpl implements CacheManager {
+
+    // There's no need for concurrency here since the caches are created at build time and never modified after that.
```

Comment:
Where is the list of caches found from?

Reply:
The list of caches is built in `CacheProcessor#recordCachesBuild`.